### PR TITLE
bshultz-toast/DOC-1936

### PR DESCRIPTION
**Pull request by bshultz-toast**
Labor API updates:
-added `host: toast-api-server` to top level of spec
-updated some summaries adhering to API Style Guide (imperative tone, period deletion)
-removed the name field from the contact section
-condensed top-level description and deleted the `## Overview` markdown
-fixed erroneous syntax on an `operationId` or two
-stripped the HTML links from Objects

### CODE REVIEW

File: openapi/toast-labor-api.yaml [hunk 1]
```diff
@@ -4,22 +4,18 @@ info:
...
-    jobs, and shifts for your restaurant. The labor API is intended for software
-    engineers, managers, and technical staff who are responsible for integrating
-    third-party systems with the Toast POS system.
+    Toast labor API is a set of REST web services that you can use to manage the employees, jobs, and shifts 
+    for your restaurant. The labor API is intended for software engineers, managers, and technical staff who 
+    are responsible for integrating third-party systems with the Toast POS system.
 
-    See general information about using Toast REST APIs in the <cite>Toast API
-    Developer's Guide</cite>.
+    See <cite>Toast API
```
Comment (pdesjardins) [inline openapi/toast-labor-api.yaml:12]: Let's remove these lines entirely. The API reference is included in the same site with the developer guide, so there's no need for this reference.

File: openapi/toast-labor-api.yaml [hunk 2]
```diff
@@ -420,11 +416,11 @@ definitions:
...
   /employees:
     get:
-      summary: Get details on restaurant employees.
+      summary: Get details on restaurant employees
       description: Returns an array of
-        <a href="#/definitions/Employee">Employee</a> objects containing
+        Employee objects containing
         information about restaurant employees.
-      operationId: employeesGetInformation
+      operationId: employeesInformationGet
```
Comment (pdesjardins) [inline openapi/toast-labor-api.yaml:423]: Do we need to put "information" in there? Why not `employeesGet`? I think that would be the exact concatenation.

File: openapi/toast-labor-api.yaml [hunk 3]
```diff
@@ -497,9 +493,9 @@ paths:
           description: An unexpected internal error occurred. There is a requestId attached to this error that can be referenced by Toast.
   /employees/{employeeId}:
     get:
-      summary: Get details on a restaurant employee.
-      description: Returns an <a href="#/definitions/Employee">Employee</a> object containing information about one restaurant employee.
-      operationId: employeeGetInformation
+      summary: Get details on a restaurant employee
+      description: Returns an Employee object containing information about one restaurant employee.
+      operationId: employeeInformationGet
```
Comment (pdesjardins) [inline openapi/toast-labor-api.yaml:498]: Would this be `employeesEmployeeId`?

File: openapi/toast-labor-api.yaml [hunk 4]
```diff
@@ -993,7 +989,7 @@ paths:
             `requestId` attached to this error that can be referenced by Toast
             support.
     put:
-      summary: Updates existing schedule shift record.
+      summary: Update existing schedule shift record
```
Comment (pdesjardins) [inline openapi/toast-labor-api.yaml:992]: Update _an_ existing schedule shift...

???

File: openapi/toast-labor-api.yaml [hunk 5]
```diff
@@ -1037,7 +1033,7 @@ paths:
         '500':
           description: An unexpected internal error occurred. There is a requestId attached to this error that can be referenced by Toast.
     delete:
-      summary: Deletes existing schedule shift record.
+      summary: Delete existing schedule shift record
```
Comment (pdesjardins) [inline openapi/toast-labor-api.yaml:1036]: Delete an existing schedule shift record

?

File: openapi/toast-labor-api.yaml [hunk 6]
```diff
@@ -1146,7 +1142,7 @@ paths:
         type: string
         required: true
     post:
-      summary: Adds external identifier to existing job.
+      summary: Add external identifier to existing job
```
Comment (pdesjardins) [inline openapi/toast-labor-api.yaml:1145]: Add an external identifier to an existing job

??

File: openapi/toast-labor-api.yaml [hunk 7]
```diff
@@ -1300,8 +1296,8 @@ paths:
           description: An unexpected internal error occurred. There is a requestId attached to this error that can be referenced by Toast.
   /timeEntries/{timeEntryId}:
     get:
-      summary: Get information about one shift's events.
-      description: Returns an <a href="#/definitions/TimeEntry">TimeEntry</a>
+      summary: Get information about one shift's events
```
Comment (pdesjardins) [inline openapi/toast-labor-api.yaml:1299]: Do some of these need `operationId` fields?

**Comment by bshultz-toast** [comment]
Fixed all above issues as well as all erroneous `operationId`s

File: openapi/toast-labor-api.yaml [hunk 8]
```diff
@@ -4,22 +4,15 @@ info:
...
+    for your restaurant. The labor API is intended for software engineers, managers, and technical staff who 
+    are responsible for integrating third-party systems with the Toast POS system.
 
-    See general information about using Toast REST APIs in the <cite>Toast API
-    Developer's Guide</cite>.
-
-    contact:
-    name: Toast API Support
-    email: developer-support@toasttab.com
+    contact: Toast developer support
```
Comment (jeanninecullen-toast) [inline openapi/toast-labor-api.yaml:12]: I think the indent needs to be changed so it doesn't read as description content.

File: openapi/toast-labor-api.yaml [hunk 9]
```diff
@@ -9,7 +9,8 @@ info:
     for your restaurant. The labor API is intended for software engineers, managers, and technical staff who 
     are responsible for integrating third-party systems with the Toast POS system.
 
-    contact: Toast developer support
+    contact: 
+      name: Toast developer support
```
Comment (bshultz-toast) [inline openapi/toast-labor-api.yaml:13]: @jeanninecullen-toast how about this and to make it consistent anyway?

**Review by jeanninecullen-toast — APPROVED** [review]
Changes look good to me.

**Review by jeanninecullen-toast — COMMENTED** [review]
"Add _an_ identifier for _an_ active employee"?

File: openapi/toast-labor-api.yaml [hunk 10]
```diff
@@ -1300,12 +1294,12 @@ paths:
           description: An unexpected internal error occurred. There is a requestId attached to this error that can be referenced by Toast.
   /timeEntries/{timeEntryId}:
     get:
-      summary: Get information about one shift's events.
-      description: Returns an <a href="#/definitions/TimeEntry">TimeEntry</a>
+      summary: Get information about one shift's events
+      description: Returns an TimeEntry
```
Comment (pdesjardins) [inline openapi/toast-labor-api.yaml:1298]: Minor: `Returns a TimeEntry object`? Your PR didn't cause this problem, so don't worry about fixing it.

**Review by pdesjardins — APPROVED** [review]
Looks good! PR passes build tests. 👍

**Pull request merged**